script.js

```diff
@@ -0,0 +1,93 @@
+const prompt = require("prompt-sync")();
+function computerSelection() {
+    const gameChoice = ["Rock" , "Paper" , "Scissors"];
+    let choices = Math.floor(Math.random() * gameChoice.length);
+    console.log(`computer choice ${gameChoice[choices]}`);
+    return gameChoice[choices];
+
+}
+// computerSelection();
+
+function playerSelection() {
+    let input = prompt("Enter Your choice game: Rock, Paper, Scissors:");
+    if (input === "Rock") {
+        console.log("You choose Rock")
+        return "Rock";
+    } else if (input === "Paper") {
+        console.log("You choose Paper");
+        return "Paper";
+    }else if (input === "Scissors") {
+        console.log("You choose Scissors");
+        return "Scissors";
+    } else {
+        console.log("Choose a valid game type");
+        return input;
+    }
+
+}
+
+
+function playRound(playerSelection, computerSelection) {
+    let input = prompt("Enter Your choice game: Rock, Paper, Scissors");
+    if (playerSelection === "Rock" && computerSelection === "Paper") {
```

Comment:
Why are you having just a single win case here? there are three win cases and three lose cases which are all supposed to be captured here in the playround